src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -48,15 +54,12 @@ export default class SidebarMenu extends React.Component {
           <SectionLinks>
             {sidebar.map(
               ({ name, files = [], labels = {}, indexFile }, index) => {
-                const isSectionActive = currentSection === index
+                const isSectionActive = currentSection === index;
```

Comment:
please, enable the pre-commit hook and/or run prettier manually. We don't use `;` and there are other places that do not match the style guide

Reply:
Used standard JS syntax, please provide a link to style guide

Reply:
This is the link: https://dvc.org/doc/user-guide/contributing-documentation. There is a command in package.json to run the `prettier` -`npm run prettier-js`. It will be part of the pre-commit hook soon.

Reply:
fixed